navigation/navigation-compose/src/androidMain/kotlin/androidx/navigation/compose/internal/NavHostInternals.android.kt

```diff
@@ -34,5 +34,3 @@ internal actual fun PredictiveBackHandler(
 ) {
     PredictiveBackHandler(enabled, onBack)
 }
-
-internal actual typealias DefaultNavTransitions = StandardDefaultNavTransitions
```

Comment:
Probably accidentally reverted during the merge. Please revert if it's still needed